src/platform/logging/outputChannelLogger.ts

```diff
@@ -8,7 +8,7 @@ const format = require('format-util') as typeof import('format-util');
 
 function formatMessage(level: string | undefined, message: string, ...data: Arguments): string {
     return level
-        ? `[${level.toUpperCase()} ${getTimeForLogging()}]: ${format(message, ...data)}`
```

Comment:
Reverting to what it used to be (the upper case and dates seems to be unnecessary & draws attention to that text when looking at the logs). Date was just taking up unnecessary space.